Week-3/Exercise-3.1/script.js

```diff
@@ -0,0 +1,22 @@
+//Given reducer method:
+function add(a, b) {
+  return a + b;
+}
+let cache = new Map();
```

Comment:
The cache here becomes a global variable. This pollutes global space. It should be with in the memoize function

Reply:
resolved and committed